src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/defaultContext.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/Test.java b/Test.java
```

Comment:
What for this file ?

All inputs (config, patch, input java files,... ) Should be separated into special folder and should be used by individual test run.

Reply:
removed